react/Autosuggest/Autosuggest.js

```diff
@@ -81,14 +81,16 @@ export default class Autosuggest extends Component {
       }
     };
 
+    const suggestionsContainer = document.getElementById(containerProps.id);
```

Comment:
It's in renderSuggestionsContainer method so I don't need it here, technically....but that's a good point, need to check